glide.yaml

```diff
@@ -7,7 +7,7 @@ import:
 - package: github.com/giantswarm/certificatetpr
   version: master
 - package: github.com/giantswarm/operatorkit
-  version: master
+  version: kubernetes-1.7
```

Comment:
Here's the important change. I want to proceed with operatorkit changes to master. I created a branch kubernetes-1.7 from current master. After migration is done we can switch to master again.